CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.8.2
```

Comment:
The last release was 1.7.2. Shouldn't this be 1.8.0? Or, perhaps, 1.7.3?

Reply:
oh, yes, sure, it's `1.8.0` thanks Ewan

Reply:
fixed